packages/block-editor/src/store/defaults.js

```diff
@@ -131,6 +132,12 @@ export const SETTINGS_DEFAULTS = {
 		{ slug: 'full', name: __( 'Full Size' ) },
 	],
 
+	lineHeight: [
```

Comment:
If the theme doesn't provide any, which defaults do we want here?

Reply:
@jasmussen @kjellr @mapk this PR implements presets for line-height, as per https://github.com/WordPress/gutenberg/issues/23111 I was wondering if you had any suggestions for line-height defaults.

Reply:
1. Do we need a default? If we don't, I'd recommend not setting line height by default.
2. Browsers default to around 1.2. I tend to bump that up to 1.3-1.5, but that could be too opinionated for this.

Reply:
Also, I would make sure those values are unitless if they aren't already: `line-height: 1.2;`